src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_OF_ELEMENTS = 10;
+    private K[] arrayOfKeys = (K[]) new Object[MAX_NUMBER_OF_ELEMENTS];
+    private V[] arrayOfValues = (V[]) new Object[MAX_NUMBER_OF_ELEMENTS];
+    private int sizeOfStorage = 0;
```

Comment:
```suggestion
    private int sizeOfStorage;
```
we don't have to set it to 0 - 0 is the default value